hacks/start_chrooted

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env bash
+
+function cleanup(){
+    sed -i '/source \/etc\/bash\/install_on_demand/d' /host/etc/bash.bashrc 2>/dev/null 1>/dev/null
+    rm -rf /host/etc/bash/install_on_demand 2>/dev/null 1>/dev/null
+    rm -rf /host/root/dotfiles 2>/dev/null 1>/dev/null
+    rm -rf /host/root/.bashrc 2>/dev/null 1>/dev/null
+    rm -rf /host/hacks 2>/dev/null 1>/dev/null
+    rm -rf /host/nonroot/hacks 2>/dev/null 1>/dev/null
+    echo 'Bye!'
+}
+
+trap cleanup EXIT
+
+if [ "${#}" -eq 0 ] || [ "${1}" -ne "1" ]; then
```

Comment:
Do we need this check in a file called `start_chrooted`. I find it a bit confusing.

Reply:
File needs a better name (possibly `spawn-ops-pod-shell`).